error/messages.ts

```diff
@@ -3,4 +3,5 @@ export default {
   txnMempoolConflictCode18: 'txn-mempool-conflict (code 18)',
   missingInputs: 'Missing inputs',
   dustCode64: 'dust (code 64)',
+  requestFailed5XX: 'Request failed with status code 5',
```

Comment:
5? Shouldn't be 5xx or someting?

Reply:
I was thinking about it and if I would put XX on the end I would have to slice those two last characters in the place of usage, to be honest, I'm not sure what is better

Reply:
Me either, I just thought you made a typo. Probably we don't want to inform users about status code at all - simply a technical error occured, please try again

Reply:
no, I didn't make I typo. Yes that's why we catch all 5XX errors and translate to a general user-friendly message